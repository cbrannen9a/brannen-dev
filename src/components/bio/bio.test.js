import React from "react";
import { render } from "@testing-library/react";

import Bio from "./bio";

const bio = [
  {
    style: "normal",
    _key: "block1",
    _type: "block",
    markDefs: [],
    children: [
      {
        marks: [],
        text: "test text",
        _key: "mock1",
        _type: "span"
      },
      {
        marks: ["strong"],
        text: "test bold",
        _key: "mock2",
        _type: "span"
      },
      {
        marks: ["em"],
        text: "test italic",
        _key: "mock3",
        _type: "span"
      }
    ]
  }
];

describe("Bio", () => {
  test("renders to matchsnaphot", () => {
    const { container } = render(<Bio bio={bio} />);

    expect(container).toMatchSnapshot();
  });
});
