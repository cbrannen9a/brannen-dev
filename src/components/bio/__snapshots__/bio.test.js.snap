// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders to matchsnaphot 1`] = `
<div>
  <div
    class="bio"
  >
    <p>
      test text
      <strong>
        test bold
      </strong>
      <em>
        test italic
      </em>
    </p>
  </div>
</div>
`;
